### ISSUE TITLE
MNW2 node_data sorting causes multi-node wells to be listed from bottom to top

### ISSUE THREAD

**Issue by dbrakenhoff** [modflowpy/flopy#344]
When creating an MNW2 package using ztop and zbotm to define well screen locations, in the resulting *.mnw2 file, the order of the well screens is reversed, i.e. the well screen deepest relative to the surface is listed first. 

This behavior can be seen in the [MNW2 example notebook](https://github.com/modflowpy/flopy/blob/develop/examples/Notebooks/flopy3_mnw2package_example.ipynb). I used the first example as a basis to build a full model and run it. Initially, in the notebook the wells are defined in a rec-array as follows:

```
rec.array([(0, 1, 1, 9.5, 7.1, 'well1', 'skin', -1, 0, 0, 0, 1. , 2., 5., 6.2),
           (1, 1, 1, 7.1, 5.1, 'well1', 'skin', -1, 0, 0, 0, 0.5, 2., 5., 6.2),
           (2, 3, 3, 9.1, 3.7, 'well2', 'skin', -1, 0, 0, 0, 1. , 2., 5., 4.1)],
          dtype=[('index', '<i8'), ('i', '<i8'), ('j', '<i8'), ('ztop', '<f8'), ('zbotm', '<f8'), ('wellid', 'O'), ('losstype', 'O'), ('pumploc', '<i8'), ('qlimit', '<i8'), ('ppflag', '<i8'), ('pumpcap', '<i8'), ('rw', '<f8'), ('rskin', '<f8'), ('kskin', '<f8'), ('zpump', '<f8')])
```

After building the MNW2 package and getting the `mnw2.node_data` rec-array from the `mnw2` object this is the output:
```
rec.array([(0, 1, 1, 7.1, 5.1, 'well1', 'skin', -1, 0, 0, 0, 0.5, 2., 5., 0., 0., 0., 0., 0., 0, 0, 0, 6.2, 0., 0, 0., 0., 0., 0., 0., 0., 0., 0.),
           (0, 1, 1, 9.5, 7.1, 'well1', 'skin', -1, 0, 0, 0, 1. , 2., 5., 0., 0., 0., 0., 0., 0, 0, 0, 6.2, 0., 0, 0., 0., 0., 0., 0., 0., 0., 0.),
           (0, 3, 3, 9.1, 3.7, 'well2', 'skin', -1, 0, 0, 0, 1. , 2., 5., 0., 0., 0., 0., 0., 0, 0, 0, 4.1, 0., 0, 0., 0., 0., 0., 0., 0., 0., 0.)],
          dtype=[('k', '<i4'), ('i', '<i4'), ('j', '<i4'), ('ztop', '<f4'), ('zbotm', '<f4'), ('wellid', 'O'), ('losstype', 'O'), ('pumploc', '<i4'), ('qlimit', '<i4'), ('ppflag', '<i4'), ('pumpcap', '<i4'), ('rw', '<f4'), ('rskin', '<f4'), ('kskin', '<f4'), ('B', '<f4'), ('C', '<f4'), ('P', '<f4'), ('cwc', '<f4'), ('pp', '<f4'), ('pumplay', '<i4'), ('pumprow', '<i4'), ('pumpcol', '<i4'), ('zpump', '<f4'), ('hlim', '<f4'), ('qcut', '<i4'), ('qfrcmn', '<f4'), ('qfrcmx', '<f4'), ('hlift', '<f4'), ('liftq0', '<f4'), ('liftqmax', '<f4'), ('hwtol', '<f4'), ('liftn', '<f4'), ('qn', '<f4')])
```

Attempting to run a model with the resulting package results in an `MNW2 ERROR - Intervals`.

I think I've managed to establish that the "problem" is caused by line 836 in mfmnw2.py in the latest release (3.2.9):

```
            self.node_data.sort(order=['wellid', 'k'])
```

If I try calling this command on the original node_data rec-array in the notebook, the rec-array is not altered. However, in mfmnw2.py a new empty rec-array is created and then filled with data from the node_data that is passed to `flopy.modflow.ModflowMnw2`. Calling sort on this new rec-array changes the ordering and causes the deepest wells to be listed first, which in turn probably causes the file to be written incorrectly. Commenting out this line resolves the problem. That probably isn't the best solution however, as it can be necessary to sort the rec-array by `wellid`.

Not sure what a good fix would be, so any help is greatly appreciated.

**Comment by szipper** [comment]
Just wanted to say I've also observed this behavior - my simple workaround is this:
```
# for some reason, MNW2 package reverses node data when a model is loaded and rewritten
# need to flip node_data so that topmost node is listed first
mf.mnw2.node_data = np.flipud(mf.mnw2.node_data)
```

**Comment by langevin-usgs** [comment]
Fixed with 1e2a7820950ba6febf21532613a226dad39c42d0.  Thanks @aleaf-usgs!
